validation/standard/standard-versions.toml

```diff
@@ -1,14 +1,34 @@
-["op-contracts/v1.4.0"] # Fault Proofs https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.4.0
-address_manager = ""
-l1_cross_domain_messenger = "2.3.0"
-l1_erc721_bridge = "2.1.0"
-l1_standard_bridge = "2.1.0"
-optimism_mintable_erc20_factory = "1.9.0"
-optimism_portal = "3.10.0"
-system_config = "2.2.0"
-protocol_versions = "1.0.0"
-anchor_state_registry = "1.0.0"
-delayed_weth = "1.0.0"
-dispute_game_factory = "1.0.0"
-fault_dispute_game = "1.2.0"
-permissioned_dispute_game = "1.2.0"
+# Fault Proofs https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.4.0
+["op-contracts/v1.4.0"]
+address_manager = { version = "" }
+l1_cross_domain_messenger = { version = "2.3.0" }
```

Comment:
Why no `address` for this tag (as you have for the below)?

Reply:
Reached out to @mds1 for the `1.4.0` implementation addresses. Will add here.

Reply:
L1CrossDomainMessenger hasn't changed in a while, so it's the same implementation address as 1.3.0

Reply:
Added implementation addresses to the list, where relevant.